sql/src/main/java/io/crate/planner/SelectStatementPlanner.java

```diff
@@ -150,26 +131,27 @@ public Plan visitQueriedDocTable(QueriedDocTable table, Planner.Context context)
                 localMergePhase = MergePhase.localMerge(
                     context.jobId(),
                     context.nextExecutionPhaseId(),
-                    ImmutableList.of(topN, fp),
+                    ImmutableList.of(topN, fetchPhaseAndProjection.projection),
                     collectPhase.executionNodes().size(),
                     collectPhase.outputTypes()
                 );
             } else {
+                //noinspection OptionalGetWithoutIsPresent
```

Comment:
would prefer assigning orderBy to a local variable instead of adding a noinspection